nomad/local/grapl-local-infra.nomad

```diff
@@ -201,29 +237,41 @@ job "grapl-local-infra" {
   group "kafka" {
     network {
       mode = "bridge"
-      port "kafka-for-other-nomad-tasks" {
-        static = var.kafka_broker_port
+      port "kafka0-for-other-nomad-tasks" {
+        static = var.kafka0_broker_port
+      }
+      port "kafka0-for-host-os" {
+        static = var.kafka0_broker_port_for_host_os
       }
-      port "kafka-for-host-os" {
-        static = var.kafka_broker_port_for_host_os
+      port "kafka1-for-other-nomad-tasks" {
+        static = var.kafka1_broker_port
+      }
+      port "kafka1-for-host-os" {
+        static = var.kafka1_broker_port_for_host_os
+      }
+      port "kafka2-for-other-nomad-tasks" {
+        static = var.kafka2_broker_port
+      }
+      port "kafka2-for-host-os" {
+        static = var.kafka2_broker_port_for_host_os
       }
     }
 
-    task "kafka" {
+    task "kafka0" {
       driver = "docker"
 
       config {
         image = "confluentinc/cp-kafka:7.0.1"
-        ports = ["kafka-for-other-nomad-tasks", "kafka-for-host-os"]
+        ports = ["kafka0-for-other-nomad-tasks", "kafka0-for-host-os"]
       }
 
       resources {
-        memory = 2000
+        memory = 1200
       }
 
       env {
-        kafka_broker_port       = 9092 # Only used by healthcheck
-        KAFKA_BROKER_ID         = 1
+        kafka0_broker_port      = 9092 # Only used by healthcheck
```

Comment:
is this ever used?

Reply:
Yeah it's used in the healthcheck on L309.